### PR TITLE
[stablehlo] add aten.fmod.Tensor op conversion support

### PR DESCRIPTION
**Pull request by penguin-wwy**

### CODE REVIEW

File: lib/Conversion/TorchToStablehlo/Basic.cpp [hunk 1]
```diff
@@ -1833,6 +1833,37 @@ LogicalResult ConvertAtenOp<AtenRemainderTensorOp>::matchAndRewrite(
   return success();
 }
 
+// AtenFmodTensorOp
+// torch.fmod(a, b) == a - a.div(b, rounding_mode="trunc") * b
```
Comment (qingyunqu) [inline lib/Conversion/TorchToStablehlo/Basic.cpp:1837]: As your description, maybe it's a good idea to write a decompisition on torch dialect.

Comment (penguin-wwy) [reply lib/Conversion/TorchToStablehlo/Basic.cpp:1837]: For linalg, it decomposes directly to linalg dialect at a similar position. For ONNX, I'm not sure have any impact. I will try to implement it on torch decompisition to see if there are any issues.

**Review by stellaraccident — COMMENTED** [review]
Yeah, I don't think we should be unconditionally decomposing such primitives for the benefit of hlo level op sets. Fine to have some notion of optional decompositions that backends can use if helpful.

**Review by stellaraccident — APPROVED** [review]
While a value judgment, I think hlo is the outlier here, requiring a decomposition, so I would do it like this. It is a weak opinion, though, and I defer to the folks maintaining this backend.

**Pull request merged**